resources/eventing/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: eventing
 description: Kyma Component 'eventing'
-version: 1.0.0
```

Comment:
I don't get the logic here - why are you "downgrading" the version? Shouldn't it be `apiVersion: v3` and `version: 0.1.0`? Or just `version: 2.0.0`?

Reply:
Previously, version 1.0.0 is not correct. Hence we fixed it here with `0.1.0`. This is a chart that is being worked on and it is not installed anywhere right now. ApiVersion seems to be correct as per Helm [docs](https://helm.sh/docs/topics/charts/#the-apiversion-field)
> The apiVersion field should be v2 for Helm charts that require at least Helm 3.